server/src/test/java/org/opensearch/lucene/index/ShuffleForcedMergePolicyTests.java

```diff
@@ -59,13 +60,17 @@ public class ShuffleForcedMergePolicyTests extends BaseMergePolicyTestCase {
     public void testDiagnostics() throws IOException {
         try (Directory dir = newDirectory()) {
             IndexWriterConfig iwc = newIndexWriterConfig().setMaxFullFlushMergeWaitMillis(0);
-            MergePolicy mp = new ShuffleForcedMergePolicy(newTieredMergePolicy());
+            TieredMergePolicy tieredMergePolicy = newTieredMergePolicy();
```

Comment:
Thanks @kkewwei , I see the defaults for TieredMergePolicy per-tier / max-merge-at-once is `10`, may be we could just bump a number of docs?

```
int numDocs = 110 + random().nextInt(10);
```